releases/history.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## Version 1.0
+* ### Lastest Release: [v1.0.3](https://github.com/openebs/mayastor/releases/tag/v1.0.3)
+  * released: 11-Oct-2022
+* ### Patch Releases:
+  * [v1.0.1](https://github.com/openebs/mayastor/releases/tag/v1.0.1)
+  * [v1.0.2](https://github.com/openebs/mayastor/releases/tag/v1.0.2)

```

Comment:
Should v1.0.2 precede v1.0.1 ? Latest at the top, followed by others with oldest at the bottom?

Reply:
I'll be honest, I was copying from kubernetes.io homework.  They list in a similar fashion, using ascending order.  So I'm "following suit" with something that users will have seen elsewhere.

Reply:
However, with time to consider it, I think it is logical to go in reverse chronological order here.  I'll update.